src/twisted/internet/wxreactor.py

```diff
@@ -84,7 +84,7 @@ def _installSignalHandlersAgain(self):
             signal.signal(signal.SIGINT, signal.default_int_handler)
         except ImportError:
             return
-        self._handleSignals()
+        self._signals.install()
```

Comment:
I… think I'm willing to live with the coverage gap here. Is anyone still using wxpython 😬

Reply:
:grimacing: